Dockerfile

```diff
@@ -22,35 +22,33 @@ RUN --mount=type=cache,id=pnpm,target=/root/.local/share/pnpm/store \
     UV_LINK_MODE=copy BIN_DIR=/bin make package-tools
 
 FROM cgr.dev/chainguard/wolfi-base AS final
+
+# Setup postgres and postgres-vector, although in production setup we don't really need standalone postgres
+RUN apk add postgresql build-base git postgresql-dev
+RUN git clone --branch v0.8.0 https://github.com/pgvector/pgvector.git && \
```

Comment:
darren mentioned this fleetingly when the feasibility of this was discussed. i should have passed it along sooner. he said you could just get it via from, like
```
FROM pgvector/pgvector:pg17
```
or whatever. i think that would be a little cleaner and more official feeling?